### PR TITLE
cli: Show plot count in `chia plotnft show`

### LINKED ISSUE #7555: # of plots in plotnft
### Discussed in https://github.com/Chia-Network/chia-blockchain/discussions/7477

<div type='discussions-op-text'>

<sup>Originally posted by **jayhohoho2019** July 13, 2021</sup>
Hi, in v1.2.0 and 1.2.1, neither "chia farm summary" or "chia plotnft show" shows the # of plotnft plots in the pool.  But that info is available in the GUI under the Pool tab, and it refreshes.  Where is that info coming from? Which CLI command provides that? 

Thanks. </div>

### PR DESCRIPTION
**Pull request by mariano54**
Also removes unused argument `unconfirmed_transactions`

### CODE REVIEW

**Review by xdustinface — DISMISSED** [review]
Looks good, one nit.

File: chia/cmds/plotnft_funcs.py [hunk 1]
```diff
@@ -151,16 +154,20 @@ async def pprint_pool_wallet_state(
...
     address_prefix = config["network_overrides"]["config"][config["selected_network"]]["address_prefix"]
     summaries_response = await wallet_client.get_wallets()
     wallet_id_passed_in = args.get("id", None)
+    plot_counts: Counter = Counter()
     try:
         pool_state_list: List = (await farmer_client.get_pool_state())["pool_state"]
+        harvesters = await farmer_client.get_harvesters()
+        for d in harvesters["harvesters"]:
+            for plot in d["plots"]:
+                plot_counts[hexstr_to_bytes(plot["pool_contract_puzzle_hash"])] += 1
```
Comment (emlowe) [inline chia/cmds/plotnft_funcs.py:170]: If pool_contract_puzzle_hash is not in the pool dict, does this throw? is that what we want? what happens with OG plots?

Comment (xdustinface) [inline chia/cmds/plotnft_funcs.py:170]: @emlowe Good point, should probably do something like below as `pool_contract_puzzle_hash` is always in there but its `None` for OG plots which will cause `hexstr_to_bytes` to fail. 
```suggestion
                if plot["pool_contract_puzzle_hash"] is not None:
                    plot_counts[hexstr_to_bytes(plot["pool_contract_puzzle_hash"])] += 1
```

Comment (mariano54) [reply chia/cmds/plotnft_funcs.py:170]: Thank you for finding that

**Comment by mariano54** [comment]
Tested and works now

**Review by xdustinface — APPROVED** [review]
Should be good now but having the `get` with `None` as default seems a bit redundant.

**Comment by mariano54** [comment]
> Should be good now but having the `get` with `None` as default seems a bit redundant.

That's just in case the farmer returns without that field

**Comment by BassProulx** [comment]
Is it just me?
I just updated to the latest version 1.2.3.dev0 and `chia plotnft show` still does not show plot count...

**Comment by baramuse** [comment]
> Is it just me?
> I just updated to the latest version 1.2.3.dev0 and `chia plotnft show` still does not show plot count...

me neither

**Pull request merged**